### DISCUSSION TITLE
Anzeige des Verfassers? aktuell n/a

### DISCUSSION THREAD

**Discussion by git-tec** [thorsten/phpMyFAQ#2674] [Q&A] [answered]
Ich habe in der Version 3.2.1 das Problem, dass in der Ansicht des Verfassers nur "n/a" steht.

Kann ich das irgendwo umstellen?

![grafik](https://github.com/thorsten/phpMyFAQ/assets/136434874/bbae37dc-93bf-4407-8883-da46560436fb)

**Comment by thorsten** [comment, marked as answer]
Das kann jeder User selbst in seinem User Control Panel einstellen.
